dsHdmiIn.c

```diff
@@ -0,0 +1,169 @@
+/*
+ * If not stated otherwise in this file or this component's LICENSE file the
+ * following copyright and licenses apply:
+ *
+ * Copyright 2017 RDK Management
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+
+#include <stdio.h>
+#include <string.h>
+#include "dsHdmiIn.h"
+
+
+
+dsError_t dsHdmiInInit (void)
+{
+    // no hdmi in, do nothing
+    return dsERR_NONE;
```

Comment:
Since device is not having HDMI-In supportl shouldn't this be returning https://github.com/rdkcentral/rdk-halif-device_settings/blob/7c8542e13d69fc6883b0030befcfb90035cdbe1c/include/dsHdmiIn.h#L114 (dsERR_GENERAL)